### ISSUE TITLE
Unable to stand on grabbable platforms

### ISSUE THREAD

**Issue by AlbeyAmakiir** [divVerent/aaaaxy#25]
Running on Android, acquired from F-Droid.

I have gotten to a room that gives you gloves to hold platforms, but while I can grab and place the platform in that room, I can't stand on it. I fall straight through. It seems clear that you're intended to stand on them, since there's an exit at the top of the room that can only be reached by doing so.

**Comment by divVerent** [comment]
You can open up that exit by holding the platform and letting it rise to the switch. Standing on these things becomes available at a later time in the game.

**Comment by AlbeyAmakiir** [comment]
Ah, interesting. I thought the button just made the platform reappear, since I could press it without the platform and it always reappeared when I did.

Like the other issue I made, perhaps there's a way to communicate that better? The further something is from the current standards in game language, the more it needs to be flagged. And platforms are pretty universally stand-on-able. Not sure how best to communicate that, though.

Like, for example, the walkway that disappears when you jump is weird, but well-flagged. It tricks you, but then clearly communicates what the trick was and how intentional it was, so I knew it wasn't a bug or anything. Interesting design problem.
